src/QueryEditor/QueryOptions.tsx

```diff
@@ -213,7 +213,10 @@ export const QueryOptions = (props: Props): JSX.Element => {
       </InlineFieldRow>
       {!props.hideGroupBy && (
         <InlineFieldRow>
-          <InlineLabel width={labelWidth} tooltip="Add all tags to group by">
+          <InlineLabel 
+            width={labelWidth} 
+            tooltip="Group measurement values as a field (e.g. column/trend) per tag (e.g. status)"
```

Comment:
column/trend gives more confusion. Group as a field also seems wierd

```suggestion
            tooltip="Group values per tag (e.g. status)"
```